docs/user-guide/installation/windows.md

```diff
@@ -18,16 +18,16 @@ _Note_: Keep in mind that your CMake version should be newer than the boost vers
 
 ### Set up your Environment
 
-**1.** Install Microsoft Visual C++ 2019 or newer (2017 may work, but is no longer tested by CI builds) [MSVC](https://visualstudio.microsoft.com/)
+**1.** Install Microsoft Visual C++ 2019 or newer that supports C++20 [MSVC](https://visualstudio.microsoft.com/)
 
-**2.** Install [Boost](https://dl.bintray.com/boostorg/release/1.74.0/binaries/)
-1.67 or later. For CMake to detect it automatically either
+**2.** Install [Boost](https://dl.bintray.com/boostorg/release/1.84.0/binaries/)
+1.74 or later. For CMake to detect it automatically either
```

Comment:
Earlier it says Boost 1.75 or newer -- is the minimum not 1.73 like it is with Linux?